File: BeerViewer/Models/Raw/kcsapi_powerup.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BeerViewer.Models.Raw
{
	public class kcsapi_powerup
	{
		public int api_powerup_flag { get; set; }
		public kcsapi_ship2 api_ship { get; set; }
		public kcsapi_deck[] api_deck { get; set; }
	}
}

File: WindowFrame/src/KanColle/Raw/kcsapi_powerup.ts
// tslint:disable:class-name
import { kcsapi_ship2 } from "@KC/Raw/kcsapi_ship";
import { kcsapi_deck } from "@KC/Raw/kcsapi_deck";

export interface kcsapi_powerup {
	api_powerup_flag: number;
	api_ship: kcsapi_ship2;
	api_deck: kcsapi_deck[];
}

export interface kcsapi_req_kaisou_powerup {
	api_id_items: string;
}
